perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -51,6 +51,15 @@ around open311_extra_data_include => sub {
             $_->{value} .= "\n\nSkanska CSC ref: $ref->{value}" if $ref;
         }
     }
+    if ( $row->contact->email =~ /Bartec/ ) {
+        my $address = $row->geocode->{resourceSets}->[0]->{resources}->[0]->{address};
```

Comment:
This should check that `$row->geocode` exists first (~10% of reports have a NULL geocode field which is quite surprising!)